mochify/index.js

```diff
@@ -14,11 +15,16 @@ exports.mochify = mochify;
 async function mochify(options = {}) {
   const config = await loadConfig(options);
 
+  const validation_error = validateConfig(config);
+  if (validation_error) {
+    throw validation_error;
+  }
```

Comment:
Why is `validateConfig` not throwing the errors directly?

Reply:
Looks like I've just been writing too much Go lately. I'll change this as it makes sense for this to behave like everything else (which is throwing to stop execution).